app/assets/javascripts/wizard/components/homepage-preview.js

```diff
@@ -21,8 +21,8 @@ export default createPreviewComponent(659, 320, {
     };
   },
 
-  paint(ctx, colors, font, width, height) {
-    this.drawFullHeader(colors, font);
+  paint(ctx, colors, font, headingFont, width, height) {
```

Comment:
Wow we really use this method signature over and over don't we? It might be a good follow up idea to convert this to some kind of object.